### DISCUSSION TITLE
Start talos node in qemu

### DISCUSSION THREAD

**Discussion by semmisha** [siderolabs/talos#7840] [Q&A]
Hello, i have a dumb question, how can i start qemu talos vm after "talosctl shutdown"
In virsh i cant see any vms

**Comment by smira** [comment]
Should be pretty much same as any other OS, Talos doesn't do anything different. 

`shutdown` by default tries to 'power off'.

**Comment by semmisha** [comment]
Let me describe more specific: 
After creating local cluster "talos create --provisioner=qemu" , talos create ~/.talos/cluster/defult-cluster folder, that hold vms configuration. After shutting down all nodes by "talosctl shutdown" i dont have any instrument to start nodes, because virsh dont recongnize vms located in ~/.talos/cluster/defult-cluster 

Maybe i should start vm by another tool? 

Tnahks for quick reply =)

**Reply by smira** [reply]
`talosctl cluster create` is more of a development tool, and it doesn't use/integrate with `libvirt`. 

You can perfectly start the machines yourself using `libvirt`, generating machine config, and following the guides on how to install and run Talos.

> You can still restart the shut down VMs created with `talosctl cluster create` by connecting to QEMU monitor socket and issuing `q` command.

**Reply by anchavesb** [reply]
@semmisha did you manage to find a way to start the Qemu images?

**Reply by smira** [reply]
If you run `talosctl cluster create`, you are limited to what it does/supports, and it's more of a development tool.

Quick reboot should help specifically to revive the node which was shut down: https://www.talos.dev/v1.9/advanced/developing-talos/#quick-reboot

All the above applies to `talosctl cluster create`, and like I said above you can use `virsh` or any other VM solution to run Talos.
